src/WebView.php

```diff
@@ -164,7 +167,7 @@ public function endPage(bool $ajaxMode = false): void
             sprintf(self::PLACEHOLDER_BODY_END, $this->getPlaceholderSignature()) => $this->renderBodyEndHtml($ajaxMode),
         ]);
 
-        $this->clear();
+        $this->state->clear();
```

Comment:
Suggest leave `$this->clear()` as is, and then discuss about this in another issue/PR.

Reply:
Discussed it internally. It's OK as in the PR.